auto/unix

```diff
@@ -533,7 +533,6 @@ ngx_feature_libs=
 ngx_feature_test="setsockopt(0, IPPROTO_TCP, TCP_FASTOPEN, NULL, 0)"
 . auto/feature
 
-
```

Comment:
:-1: 

Please always your modifications before sending them, these unwanted modifications are easy to spot.

(also, you can amend/squash your commits)

Reply:
done